BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingViewController.swift

```diff
@@ -1,32 +1,33 @@
 //
 //  OnboardingDigitalInvoiceViewController.swift
-// GiniBank
 //
-//  Created by Nadya Karaban on 21.10.20.
+//  Copyright © 2024 Gini GmbH. All rights reserved.
 //
 
 import UIKit
 import GiniCaptureSDK
 
-// swiftlint:disable implicit_getter
-final class DigitalInvoiceOnboardingViewController: UIViewController {
-    @IBOutlet var contentView: UIView!
-    @IBOutlet var topImageView: OnboardingImageView!
-    @IBOutlet var firstLabel: UILabel!
-    @IBOutlet var secondLabel: UILabel!
-    @IBOutlet weak var scrollView: UIScrollView!
-    @IBOutlet var doneButton: MultilineTitleButton!
-    @IBOutlet weak var scrollViewTopConstraint: NSLayoutConstraint!
-    @IBOutlet weak var scrollViewBottomAnchor: NSLayoutConstraint!
+protocol DigitalInvoiceOnboardingViewControllerDelegate: AnyObject {
+    func dismissViewController()
+}
 
+final class DigitalInvoiceOnboardingViewController: UIViewController {
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var topImageView: OnboardingImageView!
+    @IBOutlet private weak var firstLabel: UILabel!
+    @IBOutlet private weak var secondLabel: UILabel!
+    @IBOutlet private weak var scrollView: UIScrollView!
+    @IBOutlet private weak var doneButton: MultilineTitleButton!
+    @IBOutlet private weak var scrollViewTopConstraint: NSLayoutConstraint!
+    @IBOutlet private weak var scrollViewBottomAnchor: NSLayoutConstraint!
+
+    weak var delegate: DigitalInvoiceOnboardingViewControllerDelegate!
```

Comment:
[Suggestion] @ValentinaIancu-Gini in dismissViewController() func delegate is marked as optional, and here its force-unwrapped. What do you think about marking it here also as optional. or remove optional inside func?

Reply:
Sure I will mark it optional. Thanks for the catch!